.gitignore

```diff
@@ -4,3 +4,5 @@
 .deptrac.cache
 /**/temp
 /output/
+/.ddev/
+/.idea/
```

Comment:
These lines, files created by local tools, belong in a [global gitignore file](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer). The project's gitignore file often only lists files generated by the projects itself or its tools (like deptrac).

Reply:
While I do agree that the .idea path should always be in the global gitignore path, Committing the ddev config can actually be quite useful. This way you can document what kind of enviroment should be used for local development. And anyone who uses ddev can share the common config. This also ensures a quicker setup for new contributors.

I for example could never put .ddev in global ignore. In almost all projects I am working on it needs to be commited.